libnmstate/ifaces/ovs.py

```diff
@@ -82,16 +86,16 @@ def port(self):
         return port
 
     def gen_metadata(self, ifaces):
-        for port_name in self.port:
-            port_iface = ifaces[port_name]
-            port_config = _lookup_ovs_port_by_interface(
-                self.port_configs, port_iface.name
+        for ovs_iface_name in self.port:
+            ovs_iface = ifaces.all_kernel_ifaces[ovs_iface_name]
```

Comment:
Why is this looking in `all_kernel_ifaces`? OVS interface should be in `all_user_space_ifaces`. Am I missing something? Thanks!

Reply:
All the port(ovs internal interface or ovs system interface) of OVS bridge has kernel representative, hence we are searching in `all_kernel_ifaces`.

Reply:
Oh, I understand. Thanks!